### ISSUE TITLE
cannot import name 'ValidationError' from 'openapi_spec_validator.exceptions'

### ISSUE THREAD

**Issue by t2y** [t2y/openapi-ext-tools#1]
How to reproduce

```bash
$ openapi-spec-cli --help
...
  File "path/to/openapi-ext-tools/openapi/spec/ext/cli/main.py", line 9, in <module>
    from openapi_spec_validator.exceptions import ValidationError
ImportError: cannot import name 'ValidationError' from 'openapi_spec_validator.exceptions' (path/to/openapi_spec_validator-0.5.1-py3.9.egg/openapi_spec_validator/exceptions.py)
```

**Comment by t2y** [comment]
The openapi-spec-validator dropped compatibility.

* https://pypi.org/project/openapi-spec-validator/

**Comment by t2y** [comment]
current versions are here.

```
openapi-schema-validator==0.3.4
openapi-spec-validator==0.5.1
```

**Comment by t2y** [comment]
We should use `OpenAPISpecValidatorError`.

```python
class OpenAPISpecValidatorError(Exception):
    pass
```

* https://github.com/p1c2u/openapi-spec-validator/blob/master/openapi_spec_validator/exceptions.py

**Comment by t2y** [comment]
released as 0.5.1.

* https://pypi.org/project/openapi-ext-tools/
